src/tests/compiler/interface/interfaceDeclarationTests.ts

```diff
@@ -26,14 +27,20 @@ describe(nameof(InterfaceDeclaration), () => {
             doTest("interface Base1 {} interface Base2 {} interface Child extends Base1, Base2 {}", "Child", ["Base1", "Base2"]);
         });
 
+        xit("should get the base when there are multiple with type parameters", () => { // TODO: issue !
+            const { sourceFile } = getInfoFromText(`interface Base1 {} interface A {} interface Child extends Base1<A>`);
+            const types = sourceFile.getInterfaceOrThrow("Child").getBaseTypes();
+            expect(types).not.to.be.empty;
+        });
```

Comment:
Not an issue because there's a compile error here. Needs to be `interface Base1<T>`.